tests/integration/components/light-table-test.js

```diff
@@ -189,3 +189,29 @@ test('passed in components can have computed properties', function(assert) {
 
   assert.equal(this.$('.custom-row.is-active').length, 0, 'none of the items are active');
 });
+
+test('onScroll', function(assert) {
+  let done = assert.async();
+  let table = new Table(Columns, createUsers(10));
+
+  this.setProperties({
+    table,
+    currentScrollY: 0,
+    onScroll() {
+      assert.ok(true, 'onScroll worked');
```

Comment:
it's probably because 300 > than the total row heights combined, did you try with a smaller value (in my experience it matches).
that being said even something as `assert.ok(e.target.scrollTop > 0)` proves your point that it did scroll.
`assert.equal(e.target.scrollTop, scrollTopAmount)` verifies that it scrolled the requested amount as well as it scrolled.

if it's not too hard I would try to add it